src/ontology/mondo-edit.obo

```diff
@@ -283234,9 +283234,9 @@ xref: SCTID:725044000 {source="MONDO:equivalentTo"}
 xref: UMLS:C2752007 {source="Orphanet:263494", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:612937"}
 is_a: MONDO:0005500 {source="DC-OMIM:612937"} ! congenital disorder of glycosylation type I
 is_a: MONDO:0015286 {source="MESH:C567857", source="MONDO:0013049/inferred", source="MONDO:Redundant", source="OMIM:612937", source="Orphanet:263494/inferred"} ! congenital disorder of glycosylation
-is_a: MONDO:0016333 ! familial dilated cardiomyopathy
 is_a: MONDO:0017749 {source="Orphanet:263494"} ! disorder of multiple glycosylation
 is_a: MONDO:0018276 {source="Orphanet:263494"} ! muscular dystrophy-dystroglycanopathy
+relationship: excluded_subClassOf MONDO:0016333 {source="https://orcid.org/0000-0001-5208-3432"} ! familial dilated cardiomyopathy
```

Comment:
this is a feature of the disease, so I excluded the superclass

Reply:
It makes sense. Thank you!